### PR TITLE
client_specific_config: add support for ifconfig-ipv6-push

### PR DESCRIPTION
**Pull request by invidian**
So IPv6 can be configured as well.

### CODE REVIEW

File: templates/client_specific_config.erb [hunk 1]
```diff
@@ -7,6 +7,9 @@ iroute-ipv6 <%= iroute_ipv6 %>
 <% if ifconfig = @ifconfig -%>
 ifconfig-push <%= ifconfig %>
 <% end -%>
+<% if ifconfig_ipv6 = @ifconfig_ipv6 -%>
```
Comment (bastelfreak) [inline templates/client_specific_config.erb]: what is the purpose of this condition? Can't we just do:

```erb
<% if ifconfig_ipv6 -%>
```

Comment (invidian) [reply templates/client_specific_config.erb]: Can be, I just followed the pattern from `ifconfig` parameter.

**Comment by bastelfreak** [comment]
@invidian can you please update the code and rebase against our latest master branch?

**Comment by invidian** [comment]
Rebased.

**Comment by bastelfreak** [comment]
Thanks for the rebase. Can you please add a unit test to verify that the config file is correct and change the condition as mentioned in the inline comments?

**Comment by invidian** [comment]
Sure, sorry I didn't add tests in first place. Added now. Let me know if this is enough :)

**Comment by MasterMind2k** [comment]
Anything new on this pull request?

**Comment by invidian** [comment]
Rebased again :)

**Comment by invidian** [comment]
Cool, thanks for merging :+1: :smile:

**Pull request merged**